### DISCUSSION TITLE
Request for Vertical Monitor Support

### DISCUSSION THREAD

**Discussion by yash-garg** [domferr/tilingshell#88] [Ideas]
As the title suggests, are there any plans to add support for monitors with vertical orientation?

Currently, creating a custom layout on a vertical monitor is not possible, as the split settings remain horizontal.

Thank you for the extension; overall, it’s really great!

**Comment by domferr** [comment]
Hey @Yash-Garg thank you! Vertical monitor should be already supported. What issue are you having? Can you share a video? Thanks

**Reply by yash-garg** [reply]
It works fine @domferr, my bad I didn't see the horizontal layout thing. Working as expected, thanks!
